### PR TITLE
SDK-3514 / rename Android repo from appboy to braze-inc

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

https://github.com/braze-inc/braze-docs/pull/5018 but without any French directory changes

#### Description of Change:
rename Android repo from appboy to braze-inc
https://jira.braze.com/browse/SDK-3514

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization/swipe_to_dismiss.md [hunk 1]
```diff
@@ -15,5 +15,5 @@ channel:
 
 Disabling swipe-to-dismiss functionality is done on a per-card basis via the [`card.setIsDismissibleByUser()`][48] method. Cards can be intercepted before display using the [`ContentCardsFragment.setContentCardUpdateHandler()`][45] method.
 
-[45]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.contentcards/-content-cards-fragment/set-content-card-update-handler.html
-[48]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.enums/-card-key/index.html#285743463%2FClasslikes%2F-1725759721
+[45]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.contentcards/-content-cards-fragment/set-content-card-update-handler.html
+[48]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.enums/-card-key/index.html#285743463%2FClasslikes%2F-1725759721
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization/swipe_to_dismiss.md:19]: This link is broken. Do you know where it should be pointed towards?

File: _docs/_developer_guide/platform_integration_guides/cordova/cordova_integration.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ If you use the Cordova SDK default setup you won't have to make any new changes
...
+By default the Cordova SDK supports in-app messages with no changes. See the [Android]({{site.baseurl}}/developer_guide/platform_integration_guides/android/in-app_messaging/integration/) or [iOS]({{ site.baseurl }}/developer_guide/platform_integration_guides/ios/in-app_messaging/overview/) integration examples for information on customizing in-app messages. Furthermore, you can look at the [sample Cordova application](https://github.com/Appboy/appboy-cordova-sdk/blob/master/sample-project/www/js/index.js) or the sample [Android](https://github.com/braze-inc/braze-android-sdk) or [iOS]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/in-app_messaging/overview/) application for implementation samples.
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/cordova/cordova_integration.md:41]:
```suggestion
By default the Cordova SDK supports in-app messages with no changes. See the [Android]({{site.baseurl}}/developer_guide/platform_integration_guides/android/in-app_messaging/integration/) or [iOS]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/in-app_messaging/overview/) integration examples for information on customizing in-app messages. Furthermore, you can look at the [sample Cordova application](https://github.com/Appboy/appboy-cordova-sdk/blob/master/sample-project/www/js/index.js) or the sample [Android](https://github.com/braze-inc/braze-android-sdk) or [iOS]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/in-app_messaging/overview/) application for implementation samples.
```

File: _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md [hunk 3]
```diff
@@ -462,24 +462,24 @@ BrazeInAppMessageManager.getInstance().setCustomInAppMessageViewWrapperFactory(C
...
-[98]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-braze-in-app-message-manager/index.html#requestDisplayInAppMessage--
-[89]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-default-in-app-message-view-wrapper/index.html
+[83]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-in-app-message-operation/index.html#27659854%2FClasslikes%2F-1725759721
+[86]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage.listeners/-i-html-in-app-message-action-listener/index.html
+[87]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.inappmessage/-i-in-app-message-view-factory/index.html
+[88]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.inappmessage/-i-in-app-message-view-wrapper-factory/index.html
+[90]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.inappmessage/-i-in-app-message-view-wrapper/index.html
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md:477]: This link is broken. Do you know what it should point to?

File: _docs/_developer_guide/platform_integration_guides/android/storage.md [hunk 4]
```diff
@@ -44,6 +44,6 @@ By default, all fields are enabled. Note that without some properties, not all f
 
 Visit our [SDK Data Collection]({{site.baseurl}}/user_guide/data_and_analytics/user_data_collection/sdk_data_collection/) article to read more about the automatically collected device properties.
 
-[1]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.enums/-device-key/index.html
-[2]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.configuration/-braze-config/-builder/set-device-object-allowlist-enabled.html
-[3]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.configuration/-braze-config/-builder/set-device-object-allowlist.html
+[1]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.enums/-device-key/index.html
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/storage.md:47]: This link is broken. Do you know what it should point to?

File: _docs/_user_guide/data_and_analytics/user_data_collection/sdk_data_collection.md [hunk 5]
```diff
@@ -66,4 +66,4 @@ If you are interested in the Minimum Integration only, and you integrate with mP
 
 {% endalert %} 
 
-[1]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.enums/-device-key/index.html "Android device-level fields"
+[1]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.enums/-device-key/index.html "Android device-level fields"
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/user_data_collection/sdk_data_collection.md:69]: This link is broken. Do you know what it should point to?

File: _includes/archive/android/universal_deep_link_delegate.md [hunk 6]
```diff
@@ -93,6 +93,6 @@ class CustomNavigator : IAppboyNavigator {
 {% endtab %}
 {% endtabs %}
 
-[udl-1]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.actions/-uri-action/index.html
-[udl-2]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy.ui/-appboy-navigator/set-appboy-navigator.html
-[udl-3]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.appboy/-i-appboy-navigator/index.html
+[udl-1]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.actions/-uri-action/index.html
```
Comment (josh-mccrowell-braze) [inline _includes/archive/android/universal_deep_link_delegate.md:96]: This link is broken. Do you know what it should point to?

File: _includes/archive/android/universal_deep_link_delegate.md [hunk 7]
```diff
@@ -93,6 +93,6 @@ class CustomNavigator : IAppboyNavigator {
...
+[udl-2]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui/-appboy-navigator/set-appboy-navigator.html
```
Comment (josh-mccrowell-braze) [inline _includes/archive/android/universal_deep_link_delegate.md:97]: This link is broken. Do you know what it should point to?

File: _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md [hunk 8]
```diff
@@ -462,24 +462,24 @@ BrazeInAppMessageManager.getInstance().setCustomInAppMessageViewWrapperFactory(C
...
-[97]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.models.inappmessage/-i-in-app-message-themeable/enable-dark-theme.html
-[98]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-braze-in-app-message-manager/index.html#requestDisplayInAppMessage--
-[89]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-default-in-app-message-view-wrapper/index.html
+[83]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-in-app-message-operation/index.html#27659854%2FClasslikes%2F-1725759721
+[86]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage.listeners/-i-html-in-app-message-action-listener/index.html
+[87]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.inappmessage/-i-in-app-message-view-factory/index.html
+[88]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.inappmessage/-i-in-app-message-view-wrapper-factory/index.html
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md:476]: This link is broken. Do you know what it should point to?

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration.md [hunk 9]
```diff
@@ -565,13 +565,13 @@ setCustomBrazeNotificationFactory(null)
...
-[72]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.configuration/-braze-config/-builder/set-default-notification-channel-name.html
-[73]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.configuration/-braze-config/-builder/set-default-notification-channel-description.html
-[74]: hhttps://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.push/-braze-firebase-messaging-service/handle-braze-remote-message.html
+[70]: https://github.com/braze-inc/braze-android-sdk/blob/master/samples/firebase-push/src/main/AndroidManifest.xml "AndroidManifest.xml"
+[72]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.configuration/-braze-config/-builder/set-default-notification-channel-name.html
+[73]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.configuration/-braze-config/-builder/set-default-notification-channel-description.html
+[74]: hhttps://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.push/-braze-firebase-messaging-service/handle-braze-remote-message.html
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration.md:571]:
```suggestion
[74]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.push/-braze-firebase-messaging-service/handle-braze-remote-message.html
```

File: _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md [hunk 10]
```diff
@@ -462,24 +462,24 @@ BrazeInAppMessageManager.getInstance().setCustomInAppMessageViewWrapperFactory(C
...
-[95]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage.listeners/-i-in-app-message-manager-listener/after-in-app-message-view-closed.html
-[97]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.models.inappmessage/-i-in-app-message-themeable/enable-dark-theme.html
-[98]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-braze-in-app-message-manager/index.html#requestDisplayInAppMessage--
-[89]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-default-in-app-message-view-wrapper/index.html
+[83]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-in-app-message-operation/index.html#27659854%2FClasslikes%2F-1725759721
+[86]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage.listeners/-i-html-in-app-message-action-listener/index.html
+[87]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy.ui.inappmessage/-i-in-app-message-view-factory/index.html
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md:475]: This link is broken. Do you know what it should point to?

File: _docs/_developer_guide/platform_integration_guides/android/news_feed/integration.md [hunk 11]
```diff
@@ -144,16 +144,16 @@ To log a display of the feed, call [`Braze.logFeedDisplayed()`][6].
 
 To log an impression or click on a Card, call [`Card.logClick()`][7] and [`Card.logImpression()`][8] respectively.
 
-[36]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.models.cards/-card/get-extras.html
+[36]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.models.cards/-card/get-extras.html
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/news_feed/integration.md:147]: This link is broken. Do you know what it should point to?

File: _includes/archive/android/universal_deep_link_delegate.md [hunk 12]
```diff
@@ -93,6 +93,6 @@ class CustomNavigator : IAppboyNavigator {
...
+[udl-3]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.appboy/-i-appboy-navigator/index.html
```
Comment (josh-mccrowell-braze) [inline _includes/archive/android/universal_deep_link_delegate.md:98]: This link is broken. Do you know what it should point to?

File: _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md [hunk 13]
```diff
@@ -462,24 +462,24 @@ BrazeInAppMessageManager.getInstance().setCustomInAppMessageViewWrapperFactory(C
...
-[25]: https://github.com/Appboy/appboy-android-sdk/blob/master/android-sdk-ui/src/main/java/com/appboy/ui/inappmessage/IInAppMessageView.java
-[34]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-braze-in-app-message-manager/index.html
-[42]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-i-in-app-message-view-factory/index.html
-[45]: https://appboy.github.io/appboy-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-in-app-message-operation/index.html
+[20]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage/-i-in-app-message-animation-factory/index.html
+[21]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage.listeners/-i-in-app-message-manager-listener/index.html
+[24]: https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze.ui.inappmessage.views/-i-in-app-message-immersive-view/index.html
+[25]: https://github.com/braze-inc/braze-android-sdk/blob/master/android-sdk-ui/src/main/java/com/appboy/ui/inappmessage/IInAppMessageView.java
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/android/in-app_messaging/customization/custom_listeners.md:468]: This link is broken. Do you know what it should point to?

**Review by josh-mccrowell-braze — CHANGES_REQUESTED** [review]
There are a few broken links - probably already broken by the time they came to you - but I'm not sure where they're supposed to go. Is this something we can take care of in this PR?

**Comment by radixdev** [comment]
@josh-mccrowell-braze Can you file a new story with the broken links instead? They were indeed broken before this PR and a smaller scoped story specifically to fix those bad links would be more appropriate than lumping them in with 404 lines of unrelated URL updates.

**Comment by josh-mccrowell-braze** [comment]
@radixdev : Deal. If you accept the small suggested changes for typos and link me to your Jira page so I can submit the story, I'll merge now.

**Comment by radixdev** [comment]
https://jira.braze.com/browse/SDK-3514 is the story for this ticket and the board is SDK

**Review by Bucimis — APPROVED** [review]
+1 to fixing broken links in separate PR

@josh-mccrowell-braze would it be possible to provide a list of all broken links in the SDK docs to SDK team?

**Review by josh-mccrowell-braze — APPROVED** [review]

**Comment by josh-mccrowell-braze** [comment]
@Bucimis Opened a separate ticket with a list of the broken links here: https://jira.braze.com/browse/SDK-3532
If there's a better format, please let me know! Happy to oblige.

**Comment by Bucimis** [comment]
@josh-mccrowell-braze story looks great, thanks!

**Pull request merged**